libs/ui-lib/lib/ocm/components/clusterConfiguration/OcmClusterDetailsFormFields.tsx

```diff
@@ -114,11 +114,9 @@ export const OcmClusterDetailsFormFields = ({
     nameInputRef.current?.focus();
   }, []);
 
-  const handleExternalPartnerIntegrationsChange = React.useCallback<
-    NonNullable<OcmCheckboxFieldProps['onChange']>
-  >(
-    (value: boolean, _event: React.FormEvent<HTMLInputElement>) => {
-      const checked = Boolean(value);
+  const handleExternalPartnerIntegrationsChange = React.useCallback(
+    (isOracleSelected: boolean) => {
+      const checked = Boolean(isOracleSelected);
       setFieldValue('addCustomManifest', checked, false);
```

Comment:
The behaviour here seems a bit weird, because user may have selected the custom manifest independent of the custom manifests.

Reply:
It's the same behavior that @y0n1 implements with the checkbox for oci. WDYT @y0n1 ?

Reply:
Integrating with OCI requires adding custom manifests

Reply:
_Requires_ - as in we select custom manifests by default with OCI but the users can opt out? Or _requires_ - so we should disable the checkbox so custom manifests are definitely included?

Anyway, if it's the first case, the behavior for when users select something else after OCI will always be tricky. I agree with Celia here, it would be best if we wouldn't touch the custom manifests control and leave it up to the users. (They can always go back and remove them, after all.)

Reply:
> Requires - as in we select custom manifests by default with OCI but the users can opt out?  

Yes

Reply:
Ok, then the implementation is like we want. Maybe we can refactor it in the future